Source/WebCore/workers/service/context/ServiceWorkerThreadProxy.cpp

```diff
@@ -101,6 +106,19 @@ ServiceWorkerThreadProxy::~ServiceWorkerThreadProxy()
     m_serviceWorkerThread->clearProxies();
 }
 
+#if ENABLE(REMOTE_INSPECTOR)
+
+void ServiceWorkerThreadProxy::threadStartedRunningDebuggerTasks()
```

Comment:
NIT: i kinda prefer this be inlined so it's a bit clearer to see the order of things

Reply:
I didn't want the code to indent too many levels so I think I'll keep that for now.